Mathlib/LinearAlgebra/Multilinear/Basis.lean

```diff
@@ -11,50 +11,27 @@ import Mathlib.LinearAlgebra.Multilinear.Curry
 
 This file proves lemmas about the action of multilinear maps on basis vectors.
 
-## TODO
```

Comment:
I didn't verify closely that this if fixed: flagging this for other reviewers.

Reply:
I think that the current proof isn't worse than going through tensor products (and avoids importing tensor products).